internal/virtualcircuit/create.go

```diff
@@ -0,0 +1,172 @@
+				setParams(vrfInput, name, description, tags)
+				createInput = createVrfVirtualCircuit(vrfInput, projectID, customerIP, metalIP, md5, speed)
+			} else {
+				vlanInput := metal.NewVlanVirtualCircuitCreateInput(projectID)
+
+				setParams(vlanInput, name, description, tags)
+
+				createInput = createVlanVirtualCircuit(vlanInput, vnid, nnVlan, speed)
+			}
+
+			vc, _, err := c.Service.CreateInterconnectionPortVirtualCircuit(context.Background(), connectionID, portID).VirtualCircuitCreateInput(createInput).Execute()
+			if err != nil {
+				return fmt.Errorf("could not create Virtual Circuit: %w", err)
+			}
+
+			data := make([][]string, 1)
+
+			if vrfFlag != "" {
+				data[0] = []string{vc.VrfVirtualCircuit.GetId(), vc.VrfVirtualCircuit.GetName(), string(vc.VrfVirtualCircuit.GetType()), strconv.Itoa(int(vc.VrfVirtualCircuit.GetSpeed())), vc.VrfVirtualCircuit.GetCreatedAt().String()}
+			} else {
+				data[0] = []string{vc.VlanVirtualCircuit.GetId(), vc.VlanVirtualCircuit.GetName(), string(vc.VlanVirtualCircuit.GetType()), strconv.Itoa(int(vc.VlanVirtualCircuit.GetSpeed())), vc.VlanVirtualCircuit.GetCreatedAt().String()}
+			}
+
+			header := []string{"ID", "Name", "Type", "Speed", "Created"}
+
+			return c.Out.Output(vc.VlanVirtualCircuit, header, &data)
+		},
+	}
+	// Virtual Circuit Params
+	createVirtualCircuitCmd.Flags().StringVarP(&connectionID, "connId", "i", "", "Specify the UUID of the interconnection.")
+	createVirtualCircuitCmd.Flags().StringVarP(&portID, "portId", "p", "", "Specify the UUID of the port.")
+
+	// Common params for both VlanVirtualCircuit and VrfVirtualCircit
+	createVirtualCircuitCmd.Flags().IntVarP(&nnVlan, "vlan", "V", 0, "Adds or updates vlan  Must be between 2 and 4094")
+	createVirtualCircuitCmd.Flags().IntVarP(&speed, "speed", "s", 0, "bps speed or string (e.g. 52 - '52m' or '100g' or '4 gbps')")
+	createVirtualCircuitCmd.Flags().StringSliceVarP(&tags, "tags", "t", []string{}, "Return only the specified tags.")
+	createVirtualCircuitCmd.Flags().StringVarP(&name, "name", "n", "", "Name of the Virtual Circuit")
+	createVirtualCircuitCmd.Flags().StringVarP(&description, "description", "d", "", "Description for a Virtual Circuit")
+	createVirtualCircuitCmd.Flags().StringVarP(&projectID, "projectId", "P", "", "The project's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.")
+
+	//VlanVirtualCircuit input params
+	createVirtualCircuitCmd.Flags().StringVarP(&vnid, "vnid", "v", "", "Specify the UUID  of the VLAN.")
+
+	// VrfVirtualCircuit input params
+	createVirtualCircuitCmd.Flags().StringVarP(&md5, "md5", "M", "", "The plaintext BGP peering password shared by neighbors as an MD5 checksum")
+	createVirtualCircuitCmd.Flags().IntVarP(&peerAsn, "peerAsn", "a", 0, "The peer ASN that will be used with the VRF on the Virtual Circuit.")
```

Comment:
This should be `--peer-asn` to align with our usual format.  I'm not sure if we need a short flag for this one.